frontend/src/app/main/ui/ds/_sizes.scss

```diff
@@ -7,6 +7,8 @@
 @use "./utils.scss" as *;
 
 // TODO: create actual tokens once we have them from design
+$sz-2: px2rem(2);
+$sz-4: px2rem(4);
```

Comment:
These tokens are not needed when we remove the padding. Also, for reference:
- Size tokens are meant for things like `width`, `height`, etc. Paddings and margins are handled in spacing tokens (in `spacing.scss`)

Reply:
Got it, thanks!!